pkg/velaql/view.go

```diff
@@ -131,6 +137,96 @@ func (handler *ViewHandler) delete(ctx context.Context, cluster string, owner co
 	return handler.cli.Delete(ctx, manifest)
 }
 
+// ValidateView makes sure the cue provided can use as view.
+//
+// For now, we only check 1. cue is valid 2. `status` field exists
+func ValidateView(viewStr string) error {
+	val, err := value.NewValue(viewStr, nil, "")
+	if err != nil {
+		return errors.Errorf("error when parsing view: %v", err)
+	}
+
+	// Make sure `status` field exists
+	v, err := val.LookupValue(DefaultExportValue)
```

Comment:
We can define an `export` key instead of using the default one.

Reply:
I used `status` to keep consistent with existing views stored in ConfigMaps. Is another keyword really needed?